src/context/SWR.tsx

```diff
@@ -1,94 +1,31 @@
 import { logoutRoutes } from 'app/Unauthenticated';
 import useClient from 'hooks/supabase-swr/hooks/useClient';
+import LRU from 'lru-cache';
 import { LogoutReasons } from 'pages/Login';
 import { useNavigate } from 'react-router';
 import { ERROR_MESSAGES } from 'services/supabase';
 import { SWRConfig } from 'swr';
 import { BaseComponentProps } from 'types';
-import { getSWRSettings } from 'utils/env-utils';
 import { getPathWithParam } from 'utils/misc-utils';
 
-const swrSettings = getSWRSettings();
-
-interface IMiddleware {
-    cache: any;
-    logger: any;
-}
-
 export const singleCallSettings = {
     revalidateIfStale: false,
     revalidateOnFocus: false,
     revalidateOnReconnect: false,
 };
 
-const middleware: IMiddleware = {
```

Comment:
This stuff was for debugging cache and dedupe stuff. That is all working fine so removing.